### PR TITLE
Add HHS to geomapper util

### PR DESCRIPTION
**Pull request by chinandrew**
### Description
Currently based on the #609 branch for simpler reviewing **Rebase to main and merge after #609 is merged**

Adds the code to do HHS mapping to the geomapper util, assuming the mapping files are already made.

### Changelog
Itemize code/test/documentation changes and files added/removed.
- Add filenames and keys to load crosswalks
- Add tests
- Rename `hhs_region_number` -> `hhs`

### Fixes 
- Addresses some items discussed in #199

### CODE REVIEW

File: _delphi_utils_python/delphi_utils/geomap.py [hunk 1]
```diff
@@ -102,8 +106,10 @@ def __init__(self):
         """
         self.crosswalk_filepaths = CROSSWALK_FILEPATHS
         self.crosswalks = {
-            "zip": {"fips": None, "hrr": None, "msa": None, "pop": None, "state": None},
-            "fips": {"zip": None, "hrr": None, "msa": None, "pop": None, "state": None},
+            "zip": {"fips": None, "hrr": None, "msa": None,
```
Comment (sgsmob) [inline _delphi_utils_python/delphi_utils/geomap.py:109]: Are these always going to have `None` as the value?  Might be more scalable/futureproof to write as a dictionary comprehension:
```
"zip": {x:None for x in ["fips",...]}
```

**Review by sgsmob — APPROVED** [review]
Looks good, approving regardless of how you do or don't address my one comment.

File: _delphi_utils_python/delphi_utils/geomap.py (hunk 1 above)
Comment (chinandrew) [reply _delphi_utils_python/delphi_utils/geomap.py:109]: I think so, unless we end up preloading crosswalks sometime? I'll add your suggestion.

**Review by eujing — APPROVED** [review]
Looks good to me!

**Pull request merged**